app/modules/todo/views/item_views/todo_itemview.coffee

```diff
@@ -0,0 +1,49 @@
+Marionette = require 'backbone.marionette'
+template   = require '../../templates/item_views/todo_itemview'
+
+class TodoItemView extends Marionette.ItemView
+  template: template
+  tagName: 'li'
+  className: 'todo'
+
+  events:
+    'click input.complete': 'complete'
+    'click button.take_photo': 'takePhoto'
+
+  complete: ->
+    @model.set('completed', !@model.get('completed'))
+
+  # ------------------------------------------------------------------------- #
+  # Simple Helpers for debugging pleasure
+  #
+  debugMessage: (message) ->
```

Comment:
@flarik everyone has his own way of working with debug messages. I'd like to make it as easy as possible, offering a non-standard way of displaying debug messages is just one more new thing we are showing to developers. We are dealing with experienced developers, let them do it in their own manner.